src/radical/pilot/configs/resource_csc.json

```diff
@@ -0,0 +1,44 @@
+{
+    "mahti":
+    {
+        "description"                 : "Short description of the resource",
+        "notes"                       : "Notes about resource usage",
+
+        "schemas"                     : ["local", "batch", "interactive"],
+        "local"                       :
+        {
+            "job_manager_endpoint"    : "slurm://mahti.csc.fi/",
+            "filesystem_endpoint"     : "file://mahti.csc.fi/"
+        },
+        "batch"                       : "interactive",
+        "interactive"                 :
+        {
+            "job_manager_endpoint"    : "fork://localhost/",
+            "filesystem_endpoint"     : "file://localhost/"
+        },
+
+        "default_queue"               : "test",
+        "resource_manager"            : "SLURM",
+
+        "cores_per_node"              : 64,
+        "gpus_per_node"               : 0,
+        
+	"agent_config"                : "default",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
+        "default_remote_workdir"      : "$HOME",
+
+        "pre_bootstrap_0"             : [
+                                        "module load tykky",
+                                        "export PATH=</path/to/python/env>/bin:$PATH"
```

Comment:
it might be tricky with `tykky`, but we can remove this line
(these commands are executed first, and at his point RP will not know which env is used, but it will discover it lately and update `PATH` accordingly)

Reply:
done